docs/spec/distributing.rst

```diff
@@ -119,7 +121,7 @@ type checking as well.
 
 To have this file including with the package, maintainers can use existing packaging
 options such as ``package_data`` in ``setuptools``. For more details, see
-:ref:`the guide to providing type annotations <providing-type-annotations>`_.
+:ref:`the guide to providing type annotations <providing-type-annotations>`.
```

Comment:
This link was broken (https://typing.readthedocs.io/en/latest/spec/distributing.html#packaging-type-information)